src/styles/_catalog.scss

```diff
@@ -1,12 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, minmax(200px 1fr));
```

Comment:
you forgot comma and these style are invalid for browser
![image](https://user-images.githubusercontent.com/57598825/147564947-fe1a0626-e46f-4845-84c8-76078d88858a.png)


```suggestion
  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```